Resources/Textures/_DV/Objects/Tools/Decoys/juggernaut_decoy.rsi/meta.json

```
@@ -0,0 +1,19 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/space-wizards/space-station-14/tree/f2391cbcc932390f2f0c83cf5a057a5bf1265dc5/Resources/Textures/Objects/Tools/Decoys/juggernaut_decoy.rsi, helmet modified by @bogus_0451",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "folded"
+        },
+        {
+            "name": "jballoon",
+            "directions": 4
+        }
+    ]
+}
+
```

That's one blank line too much :^)
```suggestion
}
```